packages/usa-in-page-navigation/src/usa-in-page-navigation.twig

```diff
@@ -9,6 +9,7 @@
     data-scroll-offset="0"
     data-root-margin="0px 0px 0px 0px"
     data-threshold="1"
+  {% if minimum_heading_count %} data-minimum-heading-count={{minimum_heading_count}} {% endif %}
```

Comment:
Looks like the html templates aren't building because the attribute value isn't being quoted!

```suggestion
  {% if minimum_heading_count %} data-minimum-heading-count="{{minimum_heading_count}}" {% endif %}
```

Reply:
Resolved in [63c9b39](https://github.com/uswds/uswds/pull/6205/commits/63c9b39b3381b157426380d92a4cac2c37ab9172) 🥳